README.md

```diff
@@ -23,7 +23,15 @@ To build and compile the project, follow these steps:
 - Run the following command to build the JAR file: `mvn clean package`
 
 ## Usage
-- To start the program with the following command: `java -jar <your-project>.jar`
+The input file should contain only commands from block **Examples of syntax**.
```

Comment:
Replace `block` with `section`.
And wrap the section title with double quotes `"`.

Reply:
fix